package.json

```diff
@@ -79,7 +79,7 @@
     "@types/lodash.union": "^4.6.6",
     "@types/sarif": "^2.1.4",
     "@types/uuid": "^8.3.1",
-    "fast-glob": "^3.2.11",
+    "fast-glob": "3.3.0",
```

Comment:
Why not to use the latest minor?
P.S. you should lock the package.json

Reply:
Because the latest minor is what introduced test failures with no code changes, and because the lock file is not commited (see https://github.com/snyk/code-client/pull/43).

Reply:
True, but we should have ensured correct behaviour beforehand, rather than rely on arbitrary package updates to fix our logic for us 😅 
There's also the question of "was it really a minor update", since obviously it introduced potentially breaking changes, but that's a separate discussion, probably for the maintainers of `fast-glob`.
But it seemed sensible to lock this version that we know is correct, and we can update as needed, when new features or security fixes come up.

> If I understand correctly, it's to ensure arbitrary dep graph consistency between our tests and CLI when being published.

That is my understanding too, we want to keep our dev envs and pipelines in line with envs using the actual library which won't have the lock file.